Device.js

```diff
@@ -1,7 +1,5 @@
-var { NativeModules } = require('react-native');
-
+var { NativeModules, Dimensions } = require('react-native');
```

Comment:
'destructuring expression' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'require' is not defined.